keyboards/clueboard/2x1800/2021/max7219.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2021 Zach White <skullydazed@gmail.com>
+ * Copyright (c) 2007 Eberhard Fahle
+ *
+ *    max7219.h - A library for controling Leds with a MAX7219/MAX7221
+ *
+ *    Permission is hereby granted, free of charge, to any person
+ *    obtaining a copy of this software and associated documentation
+ *    files (the "Software"), to deal in the Software without
+ *    restriction, including without limitation the rights to use,
+ *    copy, modify, merge, publish, distribute, sublicense, and/or sell
+ *    copies of the Software, and to permit persons to whom the
+ *    Software is furnished to do so, subject to the following
+ *    conditions:
+ *
+ *    This permission notice shall be included in all copies or
+ *    substantial portions of the Software.
+ *
+ *    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ *    EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+ *    OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ *    NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ *    HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+ *    WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ *    FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ *    OTHER DEALINGS IN THE SOFTWARE.
+ */
+#pragma once
+#include "quantum.h"
+#include "spi_master.h"
+
+// Set defaults if they're not set
+#ifndef MAX7219_LOAD
+#    define MAX7219_LOAD B0
+#endif
+
+#ifndef MAX7219_CONTROLLERS
+#    define MAX7219_CONTROLLERS 4
+#endif
+
+#ifndef MAX7219_LED_INTENSITY
+#    define MAX7219_INTENSITY 1
```

Comment:
```suggestion
#    define MAX7219_LED_INTENSITY 1
```

Reply:
Fixed this and the buffer multiplier. Thanks!